### DISCUSSION TITLE
Mixcr built in reference

### DISCUSSION THREAD

**Discussion by pbpayal** [milaboratory/mixcr#1859] [General]
What's the builtin reference for Mixcr ? Is it IMGT? 

The link - "gene library" mentioned here isn't working. So wanted to check.
> The built-in reference [gene library](http://github.com/repseqio/library) of V-, D-, J- and C- segments is thoroughly compiled for every available species using the data specifically obtained from multiple dedicated sequencing runs as well as from hundreds of other experiments. At the same time MiXCR supports using of [external libraries](https://mixcr.com/mixcr/guides/external-libraries/) or even to assemble a [custom library](https://mixcr.com/mixcr/guides/create-custom-library/) from scratch.

**Comment by mizraelson** [comment]
Hi, MiXCR uses its own reference by default, not IMGT. We manually create and keep it up to date.

**Reply by pbpayal** [reply]
What are your sources for creating the reference?

**Reply by mizraelson** [reply]
It is a genome-based reference based on the latest assembly.

**Reply by pbpayal** [reply]
Latest assembly from where? 
I have an upcoming presentation. I need this information because I have been asked this question before!

**Reply by mizraelson** [reply]
For human: https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_009914755.1/
For mouse: https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001635.26/

**Reply by pbpayal** [reply]
Thank you..thats helpful.

**Reply by lrgrs** [reply]
Is any of this information regarding how the references were built recorded in the mixcr documentation? If so, can you share the link? 
Is there a way to print the built-in reference library to JSON format so that users can have a better idea of what is in it?

**Reply by mizraelson** [reply]
The library is built on genomic data. We provide an online resource with reference sequences at https://vdj.online/library.
